enterprise/server/remote_execution/copy_on_write/copy_on_write_test.go

```diff
@@ -64,6 +65,43 @@ func TestMmap_Digest(t *testing.T) {
 
 }
 
+func TestMmap_Concurrency(t *testing.T) {
+	s, _ := newMmap(t)
```

Comment:
```suggestion
	s, err := newMmap(t)
	require.NoError(t, err)
```

Reply:
The second return arg is a path, not an error